### PR TITLE
Documented the getIterator() method

### LINKED ISSUE #7184: Add docs for Process::getIterator()
Should we add docs for this method? We recently improved it adding config flags to it: https://github.com/symfony/symfony/pull/18513

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7184.

### CODE REVIEW

File: components/process.rst [hunk 1]
```diff
@@ -69,6 +69,22 @@ for new output before going to the next iteration::
         }
     }
 
+.. tip::
+
+    The Process component uses internally a PHP iterator to get the the output
```
Comment (xabbuh) [inline components/process.rst]: [...] component internally uses a [...]

Comment (xabbuh) [inline components/process.rst]: one of the two "the"  (in front of "output") must be removed

Comment (javiereguiluz) [inline components/process.rst]: Fixed! Thanks.

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**